novawallet/Common/Services/ExtrinsicService/Substrate/ExtrinsicSenderResolution/Delegation/DelegationResolutionPathFinder.swift

```diff
@@ -141,13 +156,13 @@ extension ProxyResolution {
         }
 
         func find(
-            from paths: [CallCodingPath: ProxyResolution.GraphResult]
-        ) throws -> ProxyResolution.PathFinderResult {
+            from paths: [CallCodingPath: DelegationResolution.GraphResult]
+        ) throws -> DelegationResolution.PathFinderResult {
             if let secretBasedResult = find(callPaths: paths, walletTypeFilter: { $0 == .secrets }) {
                 return secretBasedResult
             } else if let notWatchOnlyResult = find(
                 callPaths: paths,
-                walletTypeFilter: { $0 != .watchOnly && $0 != .proxied }
+                walletTypeFilter: { $0 != .watchOnly && $0 != .proxied && $0 != .multisig }
```

Comment:
I think it will be got to have calculated property for the wallet type - `isDelegated`